app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterFragment.kt

```diff
@@ -28,20 +33,36 @@ class PhoneNumberEnterFragment :
     companion object {
         const val REQUEST_KEY = "REQUEST_KEY"
         const val RESULT_KEY = "RESULT_KEY"
-        fun create(): PhoneNumberEnterFragment = PhoneNumberEnterFragment()
+
+        fun create(countryCode: CountryCode? = null, phoneNumber: String? = null): PhoneNumberEnterFragment =
```

Comment:
I would suggest to name this params and fields with intention that we can create screen with pre-filled values / default values. 
so it's not some random country code, but it's previously entered country code, or prefilled country code. 

Also I don't like the way that GeneralErrorScreen now knows about the screens that can create it. I can see two ways here how we can change it:
1. Check that views save state and when we return to the screen - all the values are restored properly (elegant way)
2. Write this values into some kind of storage and use them **here** if they are not null. It would remove code from the GeneralErrorScreen that is responsible just for passing the values back to the PhoneEnterFragment.

Please let me know what you think about this stuff. 

Lots of love
Gleb

Reply:
Check this out bro

Reply:
good work m8!